INN/umbrella-caribbean
# Homepage recent posts updates

## Changes

This pull request makes the following changes:

- Updates the styling for Largo Recent Posts so that all posts are the same height and the bottom lines and social icons stick to the bottom of the post on the same plane, instead of at differing heights

Before:
![Screen Shot 2019-12-17 at 3 02 16 PM](https://user-images.githubusercontent.com/18353636/71029854-41fab780-20de-11ea-8f97-c72f769c5a8d.png)

After:
![Screen Shot 2019-12-17 at 3 02 03 PM](https://user-images.githubusercontent.com/18353636/71029855-41fab780-20de-11ea-9bc3-40ad4d6fb841.png)

- Updates the styling for Largo Recent Posts on the homepage so they are displayed like this on desktop:
``` 
 1
----
2 | 3
```

![screencapture-caribbean-test-2019-12-17-15_00_40](https://user-images.githubusercontent.com/18353636/71029916-66569400-20de-11ea-9cc2-2556baa51bcc.png)

## Why



For #81

## Testing/Questions

Features that this PR affects:

- Largo Recent Posts styling


Questions that need to be answered before merging:

- [ ] Is it possible to break the new styles?

Steps to test this PR:

1. Add a few Largo Recent Posts widgets to the `Homepage Left` widget area with >= 3 posts
2. View them on the frontend and verify they look ok on desktop and scale as expected on smaller sizes
